lib/std/zig/render.zig

```diff
@@ -2482,7 +2482,10 @@ fn renderDocComments(ais: *Ais, tree: Ast, end_token: Ast.TokenIndex) Error!void
     }
     const first_tok = tok;
     if (first_tok == end_token) return;
-    try renderExtraNewlineToken(ais, tree, first_tok);
+
+    if (tok != 0 and token_tags[first_tok - 1] != .l_brace) {
```

Comment:
Hmm, this `!= .l_brace` check would fail if this function was used for doc comments on function arguments. Those however seem to be handled by `renderFnProto()` directly though currently.

Would it be safe to assert the previous token is not `.l_paren` for now so that if this is changed we don't forget to update this check? A comment explaining this would be in order as well.

Reply:
Adding an assertion to prevent accidental use elsewhere would definitely make sense. Should there be just a comment for the assertion or for the whole new condition as well?